x-pack/plugins/serverless_search/public/application/components/connectors_overview.tsx

```diff
@@ -112,6 +123,29 @@ export const ConnectorsOverview = () => {
       </EuiPageTemplate.Header>
       {connectorsLoading || (data?.connectors || []).length > 0 ? (
         <EuiPageTemplate.Section restrictWidth color="subdued">
+          <EuiCallOut
+            size="m"
+            title="Coming soon Elastic managed connectors."
+            iconType="pin"
+            onDismiss={() => {}}
+          >
+            <p>
+              {i18n.translate('xpack.serverlessSearch.connectorsOverview.calloutDescription', {
+                defaultMessage:
+                  'We´re actively developing Elastic managed connectors, that won´t require any self-managed infrastructure. You´ﬁll be able to handle all configuration in the UI. This will simplify syncing your data into a serverless Elasticsearch project. ',
+              })}
+            </p>
+            <EuiButton
+              data-test-subj="serverlessSearchConnectorsOverviewLinkButtonButton"
+              color="primary"
+              onClick={() =>
+                navigateToUrl(`${BASE_CONNECTORS_PATH}/${ELASTIC_MANAGED_CONNECTOR_PATH}`)
+              }
+            >
+              Give feedback
```

Comment:
This needs to be translated, but I'm not sure this is the right text for this button.